robottelo/ui/sync.py

```diff
@@ -172,10 +172,7 @@ def enable_rh_repos(self, repos_tree):
                     repo_name = repo_values['repo_name']
                     # UI is very slow here. Hence timeout is 120 seconds.
                     self.click(
-                        (strategy3, value3 % repo_name),
-                        waiter_timeout=120,
-                        scroll=True,
-                    )
+                        (strategy3, value3 % repo_name), waiter_timeout=120)
```

Comment:
I feel that we should pass the `scroll` method, otherwise there's no point in making it an optional argument. If, for whatever reason we choose not to scroll, the change would prevent us from doing so. The fact that you're defaulting to `scroll=True` allows us to solve this issue, so let's not remove the flexibility?